package.json

```diff
@@ -4,6 +4,10 @@
   "description": "An implementation of Http request signing and verifying",
   "private": false,
   "main": "lib/index.js",
+  "engines": {
+    "node": "=14 || =16 || >=18",
+    "yarn": "1.x"
```

Comment:
do we need to force people to use 1.x?

Reply:
reverted. had tryied switched to diff yarn, didnt seems restrict its usage on diff yarn version.